packages/web/src/generators/application/application.ts

```diff
@@ -217,7 +217,7 @@ export async function applicationGenerator(host: Tree, schema: Schema) {
       project: options.projectName,
       skipSerializers: true,
       setupFile: 'web-components',
-      babelJest: options.babelJest,
+      transformer: options.babelJest ? 'babel-jest' : 'ts-jest',
```

Comment:
same suggestion as above. we should update the `--babelJest` to be `transformer` so swc/jest can be an option in those projects.